types/externs.d.ts

```diff
@@ -71,7 +71,10 @@ export interface CliFlags extends Flags {
 export interface RunnerResult {
   lhr: LHResult;
   report: string|string[];
-  /** WARNING: artifacts are not guaranteed to be stable. The structure is subject to change in minor releases. */
+  /**
+   * @internal
+   * WARNING: artifacts are not guaranteed to be stable. The structure is subject to change in minor releases.
```

Comment:
woah let's not suggest there are _all_ unstable. We have a notion of public/private artifacts after all